docs/json.md

```diff
@@ -121,3 +121,36 @@ HEMMT will build the specified optionals from the `./optionals` folder.
     "particles"
 ]
 ```
+
+## headerexts
+**Type**: Array \[String\]
+
+HEMTT will apply specified header extensions to each PBO.
+
+```json
+"headerexts": [
+    "author=me"
+]
+```
+
+## modname
+**Type**: String
+
+HEMTT will use the specified mod name (without `@`) to form `@mod` folder.
+
+```json
+"modname": "my_mod"
+```
+
+## signame
+**Type**: String
+
+HEMTT will use the specified signature name as part of the full signature (`.bisign`) name.
+
+```json
+"signame": "my_custom_name"
+```
+
```

Comment:
It would be nice to put a file tree here showing the final name and place of the file for people that are less familiar with signing.